### DISCUSSION TITLE
Remove Extensions Icon from Toolbar css

### DISCUSSION THREAD

**Discussion by ChasCB** [Aris-t2/CustomCSSforFx#573] [General]
I know that you can remove the Extension Icon via "about:config" but how about using this css code in the "css/generalUI.
At the moment I have it in my_userChrome.css. Someone here may have better code. But this does work.

```
/*****************************/
/* Remove FF Extension Icon */
/*****************************/

#unified-extensions-button, #unified-extensions-button > .toolbarbutton-icon{
width: 0px !important;
padding: 0px !important;
}

```

**Comment by Aris-t2** [comment]
This project already has an option for this: https://github.com/Aris-t2/CustomCSSforFx/blob/master/current/css/buttons/unified_extensions_button_hidden.css

**Comment by ChasCB** [comment]
Thanks Aris. Apologies, I missed that one.

Also thanks for the HotFix the other day.

Cheers 🍷🐈 👬 🐈

ChasCB
